ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -0,0 +1,1413 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    text_dataset = aiplatform.TextDataset(dataset_id)
</pre><p> </p><p>You could use the <code>ds</code> var from before. Are you doing this to explicitly show how to retrieve an existing dataset?</p><p> </p><p>Feel free to omit this part and just use the 'ds' from before as most of our notebooks just do that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Yes, I think it's valuable to demonstrate how to retrieve an existing dataset, especially since it sounds like our existing notebooks <em>don't</em> do that.</p>